File: Data Structures & Algorithms/is-palindrome/submission-0.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        int left=0,right=s.size()-1;
       transform(s.begin(), s.end(), s.begin(), ::tolower);
        while(left<right)
        {
            if(!isalnum(s[left]))
            {
                left++;                
            }
            else if(!isalnum(s[right]))
            {
                right--;
            }
            else if(s[left]==s[right])
            {
                left++;
                right--;
            }
            else
            {
                return false;
            }
        }

   

        return true;
    }
};

File: Data Structures & Algorithms/climbing-stairs/submission-0.java
class Solution {
    public int climbStairs(int n) {
        
        int a,b,c=0;
        a=b=1;
        for(int i=0;i<n;i++)
        {
            c=a;
            a=a+b;
            b=c;

        }
        return c;
    }
}
